knative-operator/pkg/controller/dashboard/dashboard.go

```diff
@@ -4,35 +4,38 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"strings"
 
 	mfc "github.com/manifestival/controller-runtime-client"
 	mf "github.com/manifestival/manifestival"
 	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/common"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	servingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 var log = common.Log.WithName("dashboard")
 
 const ConfigManagedNamespace = "openshift-config-managed"
 
+const ServingDashboardPath = "deploy/resources/dashboards/grafana-dash-knative.yaml"
+const EventingBrokerDashboardPath = "deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml"
+const EventingSourceDashboardPath = "deploy/resources/dashboards/grafana-dash-knative-eventing-source.yaml"
```

Comment:
these are straight copies from upstream? the yamls?

Reply:
Yes they are slightly modified to work here and to change the UX. No functional changes.